CONTRIBUTING.md

```diff
@@ -182,3 +182,14 @@ Git tag for the full version number, like `git tag 0.21.1`, and
 push the tag to GitHub.
 
 After the release, you need to cherry-pick the release notes prep from `stable/*` to the `main` branch _and_ the stable branch for the latest minor version. For example, if the latest minor version is `0.3` and you released the patch `0.2.3`, then you need the copy the release notes `0.2.3.rst` into both `main` and `stable/0.3`. This ensures that the release notes show up for all future versions.
+
+### Cheatsheet for release process
+
+* Create branch ``jt-release-notes-x.x.x`` (ie: ``jt-release-notes-0.9.1``)
```

Comment:
Do we really need to start the branch name with `jt-*`?

Reply:
You a right! Sorry, my mistake. The rest of the branch name wasn't correct neigher. I just fixed it